### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by lkfitz**
Blog Post - Developing the CAP Research Community

### CODE REVIEW

File: app/_posts/2019-03-18-developing-the-cap-research-community.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Developing the CAP Research Community 
+author: kelly-fitzpatrick
+---
+Since launching the [Caselaw Access Project](https://case.law/) API and Bulk Data Service in October, we’ve been lucky to see a research community develop around this dataset. Today, we’re excited to share examples of how that community is using CAP data to create new kinds of legal scholarship. 
+
+Since October, we’ve seen our research community use data analysis uncover themes in the Caselaw Access Project dataset, with examples like [topic modeling U.S. supreme court cases](https://github.com/abdulra4/Topic-Modelling-US-Court-Cases) and a [quantitative breakdown](https://towardsdatascience.com/harvard-caselaw-access-project-data-by-the-numbers-4f95f5b2b30a) of our data. We’ve also seen programmatic access to the law create a space to interface with the law in new ways, from creating [data notebooks](https://www.kaggle.com/jonathanbesomi/caselaw-eda-illinois) to [text and excerpt generators](https://case.law/gallery/limericks).
```
Comment (anastasia) [inline app/_posts/2019-03-18-developing-the-cap-research-community.md]: we’ve seen our research community use data analysis **to** uncover

**Comment by anastasia** [comment]
aside from the tiny missing of "to", it looks great to me!

File: app/_posts/2019-03-18-developing-the-cap-research-community.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Outside this space, data supported by the Caselaw Access Project has found its way into a [course assignment](https://sociologicalgobbledygook.com/problem-set-1.html) to retrieve cases and citations with Python, a call to [expand the growing landscape of Wikidata](https://www.wikidata.org/wiki/Wikidata:Property_proposal/Caselaw_Access_Project_case_ID), and library guides on topics ranging from legal research to data mining. 
+
+We want to see what you build with Caselaw Access Project data! Keep us in the loop at info@case.law. Looking for ideas for getting started? Visit our [Gallery](https://case.law/gallery/) and [CAP Examples Github Repository](https://github.com/harvard-lil/cap-examples).
```
Comment (bensteinberg) [inline app/_posts/2019-03-18-developing-the-cap-research-community.md]: I would (mostly) lower-case the link titles here: `[gallery]` and `[CAP examples GitHub repository]`.

Comment (bensteinberg) [reply app/_posts/2019-03-18-developing-the-cap-research-community.md]: Or even `[CAP examples repository]`....

Comment (lkfitz) [reply app/_posts/2019-03-18-developing-the-cap-research-community.md]: Thanks! Should be all set.

**Pull request merged**